catalog/measure/catalog-measure-api/src/main/java/ddf/measure/Distance.java

```diff
@@ -124,10 +125,52 @@ public double getAs(LinearUnit unitOfMeasure) {
     }
 
     /**
-     * The Enum LinearUnit for the distance in meters
+     * Enumeration for the linear units supported by DDF.
      */
     public enum LinearUnit {
-        METER, KILOMETER, NAUTICAL_MILE, MILE, FOOT_U_S, YARD
-    }
+        METER, KILOMETER, NAUTICAL_MILE, MILE, FOOT_U_S, YARD;
+
+        static Map<String, LinearUnit> stringsToValues = new HashMap<>();
```

Comment:
The problem is that we're trying to map multiple strings to the same enum constant, so I don't think the constructor/`name()` approach would work. We can't override the `valueOf()` either since it's an auto-generated static method.

I could always use a `switch`/`case` in the `fromString()` but that doesn't seem much better than the map option to me.

Any other option you can think of?

I can always add a private method called `initializeValidValuesMap()` or add more comments if you think it'd help.